macos/xcode/FluentUI.xcodeproj/project.pbxproj

```diff
@@ -1376,13 +1376,15 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 8F931A6C22BD593300311764 /* FluentUI_unittest.xcconfig */;
 			buildSettings = {
+				MACOSX_DEPLOYMENT_TARGET = 13.0;
```

Comment:
oh bit . surprised we need to update our deployment target. should we update our readme file then too? what about package.swift?

Reply:
So since CI needs to be on macOS 13.0 for Xcode 14.3, I updated all the deployment targets to be 13 as well. But it's not absolutely necessary for tests to run. So would we prefer updating or leaving as is?